pkg-manager/plugin-commands-installation/src/import/index.ts

```diff
@@ -164,7 +165,7 @@ export async function handler (
   const manifest = await readProjectManifestOnly(opts.dir)
   const installOpts = {
     ...opts,
-    ...getOptionsFromRootManifest(manifest),
+    ...getOptionsFromRootManifest(opts.rootProjectManifest ?? manifest),
```

Comment:
This looks like a breaking change. Some users might currently store their settings in the nested manifests.

Reply:
I think the key is consistency. There are existing code that use `getOptionsFromRootManifest(opts.rootProjectManifest)` other than `manifest`. See comment below. So I think it's a bug we need to fix.

I'm up for honoring these options both from the root & project manifest level however. i.e.
```
{
...opts,
...getOptionsFromRootManifest(manifest),
...getOptionsFromRootManifest(opts.rootProjectManifest),
}
```
with the project manifest getting a higher priority than the root one.

However, it'd be a new feature (I think currently the official doc says these settings should be at the root package.json only?) and it has to be done in all usages across the codebase. Also, not to mention,  `getOptionsFromRootManifest` has to get modified to export fields that contain a non-undefined value.

Reply:
Honouring the options from both root and project is OK but only when the lockfile is not shared. Hence, this solution will be inconsistent as well.

Reply:
Would the current `...getOptionsFromRootManifest(opts.rootProjectManifest ?? manifest)` the best solution in the balance between making the code base consistent and yet not introducing any breaking change to users as the fallback is to the project manifest as today?

Reply:
no, as I said in the first message

> This looks like a breaking change. Some users might currently store their settings in the nested manifests.

It doesn't matter that a change would make something more consistent. This change is still a breaking change.

I am OK with honouring the options from both root and project.

Reply:
I've updated the PR with project manifest having a higher priority in merge. So it won't have any breaking change and consistent with the existing behavior.